packages/mpc-tls/prover/proto/prover.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+package prover;
+import "google/protobuf/descriptor.proto";
+
+
+extend google.protobuf.MethodOptions {
+  optional string zmq_type = 50000;
+}
+
+
+message TlsProofFilter {
+}
+message TlsProof {
+  string id = 1;
+  string data = 2;
+}
+message ValidationResult {
+  string id = 1;
+  bool ok = 2;
+}
+
+
+service Prover {
+  rpc subscribeToProofs(TlsProofFilter) returns (stream TlsProof) {
+    option (zmq_type) = 'sub';
+  }
+  rpc validate(TlsProof) returns (ValidationResult) {
+    option (zmq_type) = 'reply';
+  }
+}
+
+
+service CoreNodeJs {
```

Comment:
This may be erased too, my fault these things got unexplained on the example library :sweat_smile: